http-server/src/main/java/org/triplea/db/dao/lobby/games/LobbyGameDao.java

```diff
@@ -0,0 +1,60 @@
+package org.triplea.db.dao.lobby.games;
+
+import com.google.common.base.Ascii;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.triplea.db.dao.api.key.ApiKeyHasher;
+import org.triplea.domain.data.ApiKey;
+import org.triplea.http.client.lobby.game.lobby.watcher.ChatMessageUpload;
+import org.triplea.http.client.lobby.game.lobby.watcher.LobbyGameListing;
+import org.triplea.java.Postconditions;
+
+/**
+ * Game chat history table stores chat messages that have happened in games. This data is upload by
+ * game servers to the lobby and is then recorded in database.
+ */
+public interface LobbyGameDao {
+  int MESSAGE_COLUMN_LENGTH = 240;
+
+  default void insertLobbyGame(ApiKey apiKey, LobbyGameListing lobbyGameListing) {
+    final String hashedkey = new ApiKeyHasher().apply(apiKey);
+    final int insertCount =
+        insertLobbyGame(
+            lobbyGameListing.getLobbyGame().getHostName(), //
+            lobbyGameListing.getGameId(),
+            hashedkey);
+    Postconditions.assertState(
+        insertCount == 1, "Failed to insert lobby game: " + lobbyGameListing);
```

Comment:
Is this to check if `(select id from game_hosting_api_key where key = :apiKey)` did actually return a single entry?

Might be worth adding a comment to explain this.
I don't think any other place in the code actually checks the insert count

Reply:
If the subselect returns 0 elements we'll get a non-null constraint violation.

> I don't think any other place in the code actually checks the insert count

We should have at least a few. When adding a good number before I actually removed them so that I could PR the `postconditions` class at the same time.. Slightly interesting history; in general we should be checking insert count and not just blindly trusting it succeeded.